manifest.json

```diff
@@ -20,25 +20,27 @@
       "64": "images/toolbar/toolbar-icon-archive64.png"
     }
   },
- "applications": {
-   "gecko": {
-     "id": "wayback@web.archive.org",
-     "strict_min_version": "42.0"
-   }
- },
+  "applications": {
+    "gecko": {
+      "id": "wayback@web.archive.org",
+      "strict_min_version": "42.0"
+    }
+  },
   "homepage_url": "https://archive.org/",
   "permissions": [
-      "notifications",
-      "tabs",
       "activeTab",
+      "contextMenus",
+      "notifications",
       "storage",
+      "tabs",
       "webRequest",
       "webRequestBlocking",
-      "contextMenus",
-      "http://*/*",
-      "https://*/*",
-      "*://*/*",
-      "<all_urls>"
```

Comment:
@cgorringe I have just pushed a commit fixing that. Thank you !

Reply:
I was hoping we could leave out `<all_urls>` but if there is no way to work around it... I'll do some research to see. So far I'm afraid that they will reject the extension since I couldn't do much to trim the permissions list. The only other thing to try at this point is to remove `tabs` by rewriting the code to only make use of `activeTab` if that is possible, which I haven't tried yet...

Reply:
I think we have to keep `<all_urls>` because we want to enable 404 popup feature for all websites and for that we need to specify the URL filter for chrome.webRequest API https://developer.chrome.com/extensions/webRequest and add that to the permission section of manifest.json